tests/utils/fc.rs

```diff
@@ -224,3 +228,35 @@ pub fn run(
         .collect();
     Ok((stdout, stderr))
 }
+
+pub fn bitcoin_setup() -> bitcoincore_rpc::Client {
+    use bitcoincore_rpc::{Client, RpcApi};
+
+    let conf = config::TestConfig::parse();
+    let bitcoin_rpc =
+        Client::new(&format!("{}", conf.bitcoin.daemon), conf.bitcoin.get_auth()).unwrap();
+
+    // make sure a wallet is created and loaded
+    bitcoin_rpc.call::<bitcoincore_rpc::json::LoadWalletResult>(
```

Comment:
This can go back to the proper method that you implemented, if you think that is more suitable. I removed the if guards exclusively to get some more logs.

Reply:
Done in [acb7f73](https://github.com/farcaster-project/farcaster-node/pull/614/commits/acb7f736abd71ec9ff9739a28ca1d4f59c07bc85)

Reply:
Oh I made a mistake in that patch, will have to fix it!

Reply:
ah, correction: can be merged after #656 & this is addressed.